playwright.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/playwright:v1.34.0-jammy
+
+# upgrade ubuntu
+RUN apt update && apt upgrade -y
+
+# setup tzdata (required by pyenv dependencies)
+RUN apt update && DEBIAN_FRONTEND=noninteractive TZ=Gmt/UTC apt-get -y install tzdata
+
+# install pyenv dependencies
+RUN apt update && apt install -y \
```

Comment:
I would prefer to have one `RUN` command here. It feels more correct (I think that cache layers are not an issue anymore).

Reply:
so `RUN apt update && apt upgrade -y && apt install $all_packages`?

I spitted these run commands because the last one is taken from here:  https://github.com/pyenv/pyenv/wiki
My intend was to have a section that can be updated easily when the wiki page changes

Reply:
good idea! Done!